javascript-sdk/application-examples/collect_youtube_comments/collect_youtube_comments.js

```diff
@@ -0,0 +1,94 @@
+const { chromium } = require('playwright');
+const { configure, wrap } = require('agentql');
+
+const URL = 'https://www.youtube.com/';
+
+async function main() {
+  // Configure the AgentQL API key
+  configure({
```

Comment:
@rachelnabors and I have decided to remove them from the example. We will include how to set up API key in installation guide.

Reply:
This might be no so good, because some people don't read the installation guide (except the part with `npm install`), and go straight to the code examples to play with. 
What kind of content do you plan in installation guide? To set OS global env var or smth else?

Reply:
The installation guide currently contains guide on set session env var via `export`, but this is a good point. We should instruct users to set OS global env var